.github/workflows/llm_integration.yml

```diff
@@ -738,8 +738,8 @@ jobs:
         run: |
           echo -en "HF_MODEL_ID=nomic-ai/gpt4all-j" > docker_env
           mkdir -p models/test
-          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
-          serve -m test=file:/opt/ml/model/test/
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG no_code deepspeed \
+          serve -m test=file:/opt/ml/model/
```

Comment:
remove -m, we also don't need this

Reply:
if we remove this, is there a way to specify the model name as test? I initially removed this and then ran into issues with client.py expecting the model name to be `test`, but by default the model will be called `model`.

Reply:
We can definitely update client.py, but I thought this was easier

Reply:
ok, let's leave this now

Reply:
We need specify the model name here. we don't have a way to call management API without model name (this is not supported by SageMaker).